benchmarks/htap/queries/08.sql.template

```diff
@@ -25,7 +25,7 @@ where
     and c_id = o_c_id
     and c_w_id = o_w_id
     and c_d_id = o_d_id
-    and n1.n_nationkey = ascii(substr(c_state, 1, 1))
```

Comment:
Was that a bug, or a nice to look refactoring?

Reply:
its creating better plans this way. because there's now a foreign key constraint it knows how many rows there'll be.